### DISCUSSION TITLE
Ability to use @p in the "loader give" command

### DISCUSSION THREAD

**Discussion by chillfox** [BG-Software-LLC/WildLoaders#42] [Denied Feature Requests] [answered]
It would be great if `@p` worked so that we could use `loader give @p normal_loader` in a command block.

By the way, this is a fantastic plugin, thanks so much for making it.

**Comment by OmerBenGera** [comment, marked as answer]
You can just enter your account's name

**Comment by chillfox** [comment]
I had an idea for using command blocks to create a vending machine that gives them out to players. Without `@p` I am guessing the only way to do that would be creating a function that does it or a script on the server.

I currently have given all players on the server access to give themselves command blocks, but I would like to be able to put some kind of limits on it to control the impact on server performance.

**Reply by OmerBenGera** [reply]
> I had an idea for using command blocks to create a vending machine that gives them out to players. Without `@p` I am guessing the only way to do that would be creating a function that does it or a script on the server.
> 
> I currently have given all players on the server access to give themselves command blocks, but I would like to be able to put some kind of limits on it to control the impact on server performance.

Don't use a command block and instead use a plugin that can execute commands on interaction of blocks or something like that.
Command blocks are never the way to go.

**Reply by chillfox** [reply]
Thanks, I will look into it.
